### DISCUSSION TITLE
VS Code模板编译使用问题

### DISCUSSION THREAD

**Discussion by sjtu-jq** [sjtug/SJTUThesis#858] [提问]
请教一下VS Code怎么用这个模板啊？按照常规的把 .cls文件放到目录没法编译，提示 
> ! LaTeX Error: File `sjtu-math-font-newtx.def' not found.
readme里面写得太简单，不像是可以直接用在vs code里面，只能用写好的compile.bat脚本编译，不方便

**Comment by LogCreative** [comment]
使用 latexmk(xelatex) 方法编译，会通过 .latexmkrc 文件识别依赖目录。

FYI #573，以及本地使用说明的 [wiki](https://github.com/sjtug/SJTUThesis/wiki/本地使用说明)。

**Reply by sjtu-jq** [reply]
明白了，为了方便，我选择把sjtuthesis放到了texlive的 texmf-dist/tex/latex/ 文件夹 texhash安装了一下也好了
